test/ethereum_test.dart

```diff
@@ -11,31 +11,37 @@ void main() {
     'Ethereum test',
     () async {
       final rng = Random.secure();
-      final privateKey =
-          Web3.privateKeyFromMnemonic('amount zebra lecture crew select clay define faculty globe story bitter canvas');
+      final privateKey = Web3.privateKeyFromMnemonic(
+          'tiny kitchen artefact label morning axis disorder buffalo fiscal keen valid taxi');
       final privateEthCredentials = EthPrivateKey.fromHex(privateKey);
       final wallet = Wallet.createNew(privateEthCredentials, 'Hello', rng);
       debugPrint(wallet.toJson());
 
-      const apiUrl = "HTTP://127.0.0.1:7545"; //Replace with your API
+//      const apiUrl = "HTTP://127.0.0.1:7545"; //Replace with your API
+      const apiUrl =
+          "https://ropsten.infura.io/v3/96ac5dcb92d545b6a7ffc3d8af21fde0"; //Replace with your API
 
       final httpClient = Client();
       final ethClient = Web3Client(apiUrl, httpClient);
 
       final balance = await ethClient.getBalance(
         EthereumAddress.fromHex(wallet.privateKey.address.hex),
       );
+
+      debugPrint(wallet.privateKey.address.hex);
+      debugPrint(privateEthCredentials.address.hex);
       debugPrint("${balance.getValueInUnit(EtherUnit.ether)}");
-      await ethClient.sendTransaction(
```

Comment:
lets avoid commenting out the code, if we don't need it, just remove it. we can use git history in order to retrieve back the code if needed

Reply:
✅